apps/www/hooks/use-toast.ts

```diff
@@ -140,7 +140,7 @@ interface Toast extends Omit<ToasterToast, "id"> {}
 function toast({ ...props }: Toast) {
   const id = genId()
 
-  const update = (props: ToasterToast) =>
+  const update = (props: Toast) =>
```

Comment:
changed the type here as you don't need to pass the `id` when you do x.update()`